moveit_ros/visualization/CMakeLists.txt

```diff
@@ -79,6 +79,16 @@ pluginlib_export_plugin_description_file(rviz_common robot_state_rviz_plugin_des
 #  add_rostest(test/moveit_joy.test)
 #endif()
 
+ament_export_libraries(moveit_motion_planning_rviz_plugin_core)
+ament_export_libraries(moveit_motion_planning_rviz_plugin)
```

Comment:
I don't think the plugins should be exported (what's the purpose), the core libs should be sufficient.

Reply:
You're right, I removed them